src/cli/index.js

```diff
@@ -46,11 +47,20 @@ sywac
         desc: 'How long to wait (in milliseconds) before retrying when openFileLimit has been reached',
         defaultValue: 2000
     })
+    .boolean('--outputDepcache', {
+        desc: 'Whether to output for use with systemjs depcache or not',
+        defaultValue: false
+    })
     .help('-h, --help')
     .parseAndExit()
     .then(options => ({ ...options, logging: loggingMap[options.logging] }))
     .then(options => new CliOptions(options))
-    .then(cliOptions => buildAppOutput(cliOptions))
+    .then(cliOptions => {
+        if(cliOptions.outputDepcache) {
```

Comment:
Could we get a description and/or sample in the README of what this is and does?

Reply:
Added something to the README. Let me know if it's enough, or more details are needed

Reply:
That does the job nicely. Thank you